### ISSUE TITLE
False positive for react/jsx-pascal-case on namespaced components

### ISSUE THREAD

**Issue by strawbrary** [yannickcr/eslint-plugin-react#637]
In v5.2.0 I'm getting a false positive for the `react/jsx-pascal-case` rule when using namespaced components.

```jsx
<Modal show={showModal} onHide={this.closeModal}>
  <Modal.Header closeButton>
    <Modal.Title>Demo</Modal.Title>
  </Modal.Header>
</Modal>
```

It looks like it was introduced in cd5ef2b17257d10f5398c35d7fc73fdfb9a360f9.

**Comment by ljharb** [comment]
ping @evcohen

**Comment by evcohen** [comment]
sorry bout that 😢 
re: yannickcr/eslint-plugin-react/pull/638

**Comment by yannickcr** [comment]
No problem @evcohen, thanks for the very quick fix 😉
